README.rst

```diff
@@ -139,3 +139,47 @@ Celery
   k8s_worker_image_pull_policy: "{{ k8s_container_image_pull_policy }}"
   k8s_worker_image_tag: "{{ k8s_container_image_tag }}"
   k8s_worker_resources: "{{ k8s_container_resources }}"
+
+
+Amazon S3: IAM role for service accounts
+````````````````````````````````````````
+
+Django applications running on AWS typically use Amazon S3 for static and media
```

Comment:
Maybe just say "web applications" because of #26 :)

Reply:
done!